aidial_adapter_openai/app.py

```diff
@@ -26,6 +31,20 @@
 model_aliases: Dict[str, str] = json.loads(os.getenv("MODEL_ALIASES", "{}"))
 azure_api_version = os.getenv("AZURE_API_VERSION", "2023-03-15-preview")
 
+dial_use_file_storage = os.getenv("DIAL_USE_FILE_STORAGE", False)
```

Comment:
You need to parse the string to a boolean.

Otherwise, if `DIAL_USER_FILE_STORAGE=False`, then the condition `if dial_use_file_storage:` will still be successful.

You could use a helper like this:

```python
def bool_env_var(name: str, default: bool = False) -> bool:
    return (os.getenv(name, str(default)).lower() == 'true')
```

Reply:
fixed, thank you